inc/spbc-scanner.php

```diff
@@ -906,6 +906,26 @@ function spbc_scanner_page_view($direct_call = false, $page_url = false) {
 
 }
 
+function spbc_scanner_page_approve($direct_call = false) {
+
+    global $wpdb;
+
+    if( ! $direct_call ) {
```

Comment:
Why use the $direct_call parameter if the function is never called with this parameter?

Reply:
https://github.com/CleanTalk/security-malware-firewall/pull/109/commits/5a0b84cf14c79df028a55d67aace2156a6acb6cc Done